### PR TITLE
Add Header and Subheader

### PR DESCRIPTION
**Pull request by ndelarans**
image
_**Responsive**_
image
image

### CODE REVIEW

File: src/common-pages/Subheader.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+    useEffect(() => {
+        window.addEventListener('resize', handleResize);
+        return () => {
+            window.removeEventListener('resize', handleResize);
+        };
+    }, [handleResize]);
+
+    const handleMenuClick = (item: string) => {
+        setActiveItem(item);
+        window.history.pushState(null, '', `/${item}`);
```
Comment (dgop92) [inline src/common-pages/Subheader.tsx]: I belive this strategy does not work, take a look of https://mui.com/material-ui/react-link/

File: src/common-pages/Subheader.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+    };
+
+    const handleIconClick = () => {
+        setShowUserName(prev => !prev);
+    };
+
+    return (
+        <AppBar position="static" style={{ backgroundColor: '#708d8d', height: isMobile ? mobileAppBarHeight : appBarHeight }}>
+            <Toolbar style={{ justifyContent: 'space-between', height: '100%' }}>
+                <div style={{ display: 'flex', flexDirection: 'row', justifyContent: isMobile ? 'flex-start' : 'space-between' }}>
```
Comment (dgop92) [inline src/common-pages/Subheader.tsx]: Research about the sx prop and mui breakpoints, your approach is not optimal

**Review by dgop92 — CHANGES_REQUESTED** [review]
Overall is good, but there are some important things to fix

**Review by AlejandroSalas1143 — APPROVED** [review]
ok

**Pull request merged**